tests/api/tests.py

```diff
@@ -387,8 +387,11 @@ def test_bad_put(self):
         with self.assertRaises(BeanBagException):
             self.client['bad_resource'] = {}
 
-        with self.assertRaises(BeanBagException):
-            self.client.bad_resource = {}
+    def test_set_attr(self):
+        # this attr is just for PDCClient's object
+        # not wrapper to beanbag's attr
+        self.client.any_attr = "This is a test attr"
+        self.assertEqual(self.client.any_attr, "This is a test attr")
```

Comment:
No need to test this.

Reply:
I saw you and Lubomir think to add a test?  Even I don't think we should add one case for this patch, which test case should we add?